### PR TITLE
Feature/map 236/add infinity scroll to comments

### PR DESCRIPTION
**Pull request by obozhko-dev**
To use functional you should run yarn install on the frontEnd to install react-intersection-observer library.
Added separately download parent comments and downloading repleis on click
You can check it on "Білокузьминівські скелі" location, I added few comment here.

PS: on old comments it probably dont working  , because I add new field to Comments mode

### CODE REVIEW

File: frontend/src/components/BigPopup/CommentSection/CommentSection.tsx [hunk 1]
```diff
@@ -26,12 +32,30 @@ const CommentSection = () => {
...
 
+  useEffect(() => {
+    const addMoreComment = () =>
+      setTopCommentsOnPageIndex(prevState => prevState + commentStepCount);
+
+    const timerId = setTimeout(() => addMoreComment(), 2000);
+
+    return () => {
+      clearTimeout(timerId);
+    };
```
Comment (naberalgamma) [inline frontend/src/components/BigPopup/CommentSection/CommentSection.tsx]: remove this

**Review by mazursasha1990 — APPROVED** [review]
It works.

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
When a user added a comment it moves down in the range, and after uploading a new portion of comments they stand in the right way
ComensRange
.

File: frontend/src/components/BigPopup/CommentSection/CommentSection.tsx (hunk 1 above)
Comment (obozhko-dev) [reply frontend/src/components/BigPopup/CommentSection/CommentSection.tsx]: I change timeout to 500 (0,5 s ) , I Like when i see title about downloading new comment
image

If I remove timeout  - I cant see it

**Comment by obozhko-dev** [comment]
> When a user added a comment it moves down in the range, and after uploading a new portion of comments they stand in the right way ComensRange .

Fixed

**Review by Yojuks — APPROVED** [review]

**Review by naberalgamma — APPROVED** [review]

**Review by mazursasha1990 — APPROVED** [review]

**Pull request merged**